addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_channel_target.py

```diff
@@ -40,7 +41,7 @@ def __gather_extensions(channels: typing.Tuple[bpy.types.FCurve],
                         blender_object: bpy.types.Object,
                         export_settings,
                         bake_bone: typing.Union[str, None]
-                        ) -> typing.Any:
+                        ):
```

Comment:
Why did the typing go away? (Some other places too, I won't comment on them all.)